File: dart-jaguar/lib/model/main_import_deal_body.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';

part 'main_import_deal_body.jser.dart';

class MainImportDealBody {
  
  final String coluuid;
  
  final List<int> dealIDs;
  
  final String dir;
  
  final String name;
  

  MainImportDealBody(
    

{
     this.coluuid = null,  
     this.dealIDs = const [],  
     this.dir = null,  
     this.name = null 
    
    }
  );

  @override
  String toString() {
    return 'MainImportDealBody[coluuid=$coluuid, dealIDs=$dealIDs, dir=$dir, name=$name, ]';
  }
}

@GenSerializer()
class MainImportDealBodySerializer extends Serializer<MainImportDealBody> with _$MainImportDealBodySerializer {

}

File: scala-akka-http-server/src/main/scala/io/swagger/server/model/Main.importDealBody.scala
package io.swagger.server.model


/**
 * @param coluuid 
 * @param dealIDs 
 * @param dir 
 * @param name 
 */
case class Main.importDealBody (
  coluuid: Option[String],
  dealIDs: Option[List[Int]],
  dir: Option[String],
  name: Option[String]
)

